packages/bls/src/index.ts

```diff
@@ -78,18 +77,36 @@ export function aggregatePubkeys(publicKeys: BLSPubkey[]): BLSPubkey {
  * @param publicKey
  * @param messageHash
  * @param signature
- * @param domain
  */
-export function verify(publicKey: BLSPubkey, messageHash: bytes32, signature: BLSSignature, domain: Domain): boolean {
+export function verify(publicKey: BLSPubkey, messageHash: bytes32, signature: BLSSignature): boolean {
   assert(publicKey, "publicKey is null or undefined");
   assert(messageHash, "messageHash is null or undefined");
   assert(signature, "signature is null or undefined");
-  assert(domain, "domain is null or undefined");
   try {
     return PublicKey
       .fromBytes(publicKey)
-      .verifyMessage(Signature.fromCompressedBytes(signature), messageHash, domain);
+      .verifyMessage(Signature.fromCompressedBytes(signature), messageHash);
+  } catch (e) {
+    return false;
+  }
+}
+
+/**
+ * Verifies if aggregated signature is same message signed with given public keys.
+ * @param publicKeys
+ * @param messageHash
+ * @param signature
+ */
+export function verifyAggregate(publicKeys: BLSPubkey[], messageHash: bytes32, signature: BLSSignature): boolean {
+  assert(publicKeys, "publicKey is null or undefined");
+  assert(messageHash, "messageHash is null or undefined");
+  assert(signature, "signature is null or undefined");
+  try {
+    return Signature
+      .fromCompressedBytes(signature)
+      .verifyAggregate(publicKeys, messageHash);
   } catch (e) {
+    console.log(e)
```

Comment:
@wemeetagain Do you think it makes sense to introduce "debug" dependency for ssz and bls so we can debug when other people are using it?

Reply:
possibly? Curious why this would throw in the first place?
Perhaps we can just bubble up errors

Reply:
Well sometimes invalid serialization, wrong order, there is bunch of things though. Something wrong with messages, length etc

Reply:
I feel like those things could trigger an error rather than a false.

Reply:
well invalid signature order should trigger false though

Reply:
Let's release like this. Invalid signatures in spec tests are breaking everything if we remove try/catch